### PR TITLE
data_version

### LINKED ISSUE #23: Change versioning of the yeoda naming convention
The current yeoda naming convention is not alignment with our new versioning system. Please adopt the `yeoda_path`, so that it takes a new input argument `version` or `data_version`, instead of `version` and `run_num`. This argument should contain everything which is version related, i.e. the software version + run number and this needs to be properly set by the workflow or the user, not `geopathfinder`. This gives us also more freedom to work with other data sets having a different versioning scheme.

https://github.com/TUW-GEO/geopathfinder/blob/master/src/geopathfinder/naming_conventions/yeoda_naming.py#L260

Please also change the 'version_run_id' field to 'version' or 'data_version' everywhere.

### PR DESCRIPTION
**Pull request by bbauerma**
should fix #23

### CODE REVIEW

File: src/geopathfinder/naming_conventions/yeoda_naming.py [hunk 1]
```diff
@@ -299,21 +299,18 @@ def yeoda_path(root, product=None, version=None, run_num=None, datalog='datasets
...
-    if version:
-        if isinstance(version, int):
-            version = 'V' + "{:02d}".format(version)
-    if run_num:
-        run_num = 'R' + "{:02d}".format(run_num)
-    wflow = str(version) if version else ""
-    wflow += str(run_num) if run_num else ""
-    wflow = wflow if version or run_num else None
+    # define the data_version and run number folder name
+    if data_version:
```
Comment (fl0roth) [inline src/geopathfinder/naming_conventions/yeoda_naming.py:303]: If you agree I would change that to `if data_version is not None` to allow zero as an input.

Comment (bbauerma) [reply src/geopathfinder/naming_conventions/yeoda_naming.py]: done

**Review by fl0roth — APPROVED** [review]

**Comment by cnavacch** [comment]
When specifying 'logfiles' as a` datalog` it does not work as expected: 
https://github.com/TUW-GEO/geopathfinder/blob/data_version/src/geopathfinder/naming_conventions/yeoda_naming.py#L295
What I get back is a folder until the version, e.g. "/eodc/..../V1M1R1", but I would like to have  "/eodc/..../V1M1R1/logfiles".

File: src/geopathfinder/naming_conventions/yeoda_naming.py [hunk 2]
```diff
@@ -293,27 +293,27 @@ def yeoda_path(root, product=None, version=None, run_num=None, datalog='datasets
...
-    if version:
-        if isinstance(version, int):
-            version = 'V' + "{:02d}".format(version)
-    if run_num:
-        run_num = 'R' + "{:02d}".format(run_num)
-    wflow = str(version) if version else ""
-    wflow += str(run_num) if run_num else ""
-    wflow = wflow if version or run_num else None
+    # define the data_version and run number folder name
+    if data_version is 'logfiles':
```
Comment (cnavacch) [inline src/geopathfinder/naming_conventions/yeoda_naming.py:304]: I would use the `==` operator here.

**Pull request merged**